argilla/docs/index.md

```diff
@@ -1,11 +1,10 @@
 ---
-description: Argilla is a **collaboration platform for AI engineers and domain experts** that require **high-quality outputs, full data ownership, and overall efficiency**.
-hide: navigation
+description: Argilla is a collaboration tool for AI engineers and domain experts who need to build high-quality datasets for their projects.hide: navigation
```

Comment:
```suggestion
- description: Argilla is a collaboration tool for AI engineers and domain experts who need to build high-quality datasets for their projects.
- hide: navigation
```
The home page of the docs seems broken: This description appears above the title. Even with the suggestion here this isn't fixed.

Reply:
@davidberenstein1957